Source/Core/Common/Image.cpp

```diff
@@ -47,22 +48,39 @@ bool SavePNG(const std::string& path, const u8* input, ImageByteFormat format, u
   png.width = width;
   png.height = height;
 
+  size_t byte_per_pixel;
   switch (format)
   {
   case ImageByteFormat::RGB:
     png.format = PNG_FORMAT_RGB;
+    byte_per_pixel = 3;
     break;
   case ImageByteFormat::RGBA:
     png.format = PNG_FORMAT_RGBA;
+    byte_per_pixel = 4;
     break;
   default:
     return false;
   }
 
-  png_image_write_to_file(&png, path.c_str(), 0, input, stride, nullptr);
-  if (png.warning_or_error & PNG_IMAGE_ERROR)
+  // libpng doesn't handle non-ASCII characters in path, so write in two steps:
```

Comment:
This... doesn't have anything to do with ASCII, as I understand it. It's about the active Windows code page not being UTF-8, which is what Dolphin operates in exclusively. libpng is just calling fopen on the string that is passed in.